public/give.html

```diff
@@ -140,36 +145,46 @@
           font-weight: 400
         }
     </style>
-    <meta property="og:title" content="#findthemasks"/>
-    <meta property="og:url" content="https://findthemasks.com/"/>
-    <meta property="og:description" content="America’s frontline healthcare workers are treating COVID-19 patients without adequate protective gear, risking their lives! We need to find the masks. All of these masks can save lives now if you get them into the hands of healthcare workers."/>
+
+    <!-- Make sure to translate these meta tags when creating a new language version -->
+
+    <!-- English -->
+    <meta lang="en" property="og:title" content="#findthemasks"/>
+    <meta lang="en" property="og:url" content="https://findthemasks.com/"/>
+    <meta lang="en" property="og:description" content="America’s frontline healthcare workers are treating COVID-19 patients without adequate protective gear, risking their lives! We need to find the masks. All of these masks can save lives now if you get them into the hands of healthcare workers."/>
+
+    <!-- French -->
+    <meta lang="fr" property="og:title" content="#findthemasks"/>
+    <meta lang="fr" property="og:url" content="https://findthemasks.com/"/>
+    <meta lang="fr" property="og:description" content="Les travailleurs de la santé de première ligne du monde entier traitent les patients atteints de COVID-19 sans équipement de protection adéquat, risquant leur vie! Nous devons trouver les masques. Tous ces masques peuvent maintenant sauver des vies si vous les remettez entre les mains de professionnels de la santé."/>
+
     <meta property="og:image" content="https://findthemasks.com/images/n95.jpg"/>
 </head>
 
 <body>
 <main class="content">
     <div class="container" style="padding-bottom: 40px;">
       <div class="map-container" style="display: none;">
-        <h2 style="padding: 30px 0px 20px 0px;">Map of donation sites</h2>
+        <h2 style="padding: 30px 0px 20px 0px;" class='i18n' data-i18n="ftm-map-of-sites">Map of sites™</h2>
         <div id="map"></div>
       </div>
 
       <div class="locations-container" style="display: none;">
-        <h2 style="padding: 30px 0px 5px 0px; margin: 0px;">List of donation sites</h2>
-        <p style="padding-bottom: 10px">To add a donation site, update information, or request we take down information, please
-          <a href="https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform">fill in this form</a>.</p>
+        <h2 style="padding: 30px 0px 5px 0px; margin: 0px;" class='i18n' data-i18n="ftm-list-of-sites">List of donation sites</h2>
+        <p style="padding-bottom: 10px"> <span class='i18n' data-i18n="ftm-request-update">To add a donation site, update information, or request we take down information, please</span>
```

Comment:
Mostly out of curiosity. Does this library handle tags at all? Is it possible to wrap the entire sentence to have the full context rather than it being split into 2 strings?